packages/aila/src/lib/agentic-system/agents/sectionAgents/titleAgent/index.ts

```diff
@@ -0,0 +1,24 @@
+import { stringListToText } from "../../../utils/stringListToText";
+import { createSectionAgent } from "../createSectionAgent";
+import { titleInstructions } from "./title.instructions";
+import { LessonTitleSchema } from "./title.schema";
+import { titlesBySubjectKeyStage } from "./titlesBySubjectKeyStage";
+
+export const titleAgent = createSectionAgent({
+  responseSchema: LessonTitleSchema,
+  instructions: titleInstructions,
+  extraInputFromCtx: (ctx) => {
+    const { subject = "english", keyStage = "ks3" } = ctx.currentTurn.document;
```

Comment:
Do we really want defaults in here? This could quietly fail and make all quizzes english ks3